airflow/www/package.json

```diff
@@ -73,6 +73,7 @@
     "terser-webpack-plugin": "<5.0.0",
     "typescript": "^4.6.3",
     "url-loader": "4.1.0",
+    "web-worker": "^1.2.0",
```

Comment:
Is this used anywhere?

Reply:
It's a peer dependency of `elk` which we use to calculate how to render the graph